docs/volto-light-theme/installing-vlt.md

```diff
@@ -7,36 +7,62 @@ myst:
     "keywords": "Plone, Volto, Training"
 ---
 
+
 # Installing Volto Light Theme
 
-First for installing volto light theme.
+Follow the steps below to install and configure **Volto Light Theme** in your project. Volto Light Theme provides a clean and modern design with ready-to-use blocks and components.
 
-Go to your addon project. It is present in frontend/packages/volto-my-project and install volto-light-theme
+### Step 1: Install Volto Light Theme
 
+To install the **Volto Light Theme** in your project, navigate to the `frontend/packages/volto-my-project` folder and run the following command:
 
-```
+```bash
 pnpm install @kitconcept/volto-light-theme
-
 ```
 
-After installing vlt you will see this diff with some additional but this one is most important.
+### Step 2: Add Addons (Optional)
 
-```
-diff --git a/frontend/packages/volto-my-project/package.json b/frontend/packages/volto-my-project/package.json
-index e256491..373386e 100644
---- a/frontend/packages/volto-my-project/package.json
-+++ b/frontend/packages/volto-my-project/package.json
-@@ -26,7 +26,9 @@
-     "release-major-alpha": "release-it major --preRelease=alpha",
-     "release-alpha": "release-it --preRelease=alpha"
-   ,
--  "dependencies": {},
-+  "dependencies": {
-+    "@kitconcept/volto-light-theme": "^5.0.1"
-+  },
-   "peerDependencies":
-     "react": "18.2.0",
-     "react-dom": "18.2.0"
+Volto Light Theme comes with several pre-configured addons that provide basic blocks for your website. If you'd like to include them, you can add them to the `addons` section in your `package.json`.
```

Comment:
"add-on" is hyphenated wherever it is narrative text, and distinguishes it from the literal form `addon`.

Also use `{file}` for files wherever they appear.

```suggestion
Volto Light Theme comes with several pre-configured add-ons that provide basic blocks for your website. If you'd like to include them, you can add them to the `addons` section in your {file}`package.json`.
```

Reply:
done